### DISCUSSION TITLE
Filter out external packages

### DISCUSSION THREAD

**Discussion by cypher7682** [renovatebot/renovate#13899] [Request Help] [answered]
We're trying to filter out external stuff for now, so we can just manage internal dependencies. We've tried the following config but it's not playing ball. Any ideas? 

```
#...
  "packageRules": [{
    "matchPackagePattern": [".*"],
    "enabled": false
  },
  {
    "matchPackagePattern": ["ourcompanyname"],
    "allowedVersions": "<=0.0",
    "enabled": true
  }],
#...
```

**Comment by rarkins** [comment, marked as answer]
Simplify to this:

```
"packageRules": [{
  "excludePackagePatterns": ["ourcompanyname"],
  "enabled": false
  }]
```

I don't know what you intended with `allowedVersions`, but it didn't seem practical so I removed it.

**Reply by cypher7682** [reply]
Oh that works nicely. Thanks @rarkins 👍
